supervision/annotators/core.py

```diff
@@ -898,6 +901,7 @@ def annotate(
                 else custom_color_lookup,
             )
             center = (int(xy[detection_idx, 0]), int(xy[detection_idx, 1]))
+            cv2.circle(scene, center, self.radius, (0, 0, 0), self.outline_thickness)
```

Comment:
Let's only do this when `self.outline_thickness` is not 0, just like in the triangle annotator

Reply:
Yes @LinasKo, will add a condition statement here. Updating it and submiting.